doc/lua_api.txt

```diff
@@ -6909,10 +6909,29 @@ child will follow movement and rotation of that bone.
     object.
 * `set_detach()`
 * `set_bone_position([bone, position, rotation])`
-    * `bone`: string. Default is `""`, the root bone
-    * `position`: `{x=num, y=num, z=num}`, relative, `default {x=0, y=0, z=0}`
-    * `rotation`: `{x=num, y=num, z=num}`, default `{x=0, y=0, z=0}`
+	* Shorthand for `set_bone_override(bone, {position = ..., rotation = ...})` using absolute values.
+	* Supports older servers & clients (< 5.6)
+	* **Deprecated**; use `set_bone_override` instead
 * `get_bone_position(bone)`: returns position and rotation of the bone
+	* Shorthand for `get_bone_override(bone).position.vector, get_bone_override(bone).rotation.vector`
+	* **Deprecated**, use `get_bone_override` instead
+* `set_bone_override(bone, override)`
+    * `bone`: string
+    * `override`: `{ position = property, rotation = property, scale = property }` or `nil`
+        * `override = nil` (or nothing) is shorthand for `override = {}` which clears the override
+        * `property`: `{ x = ..., y = ..., z = ..., interpolation_duration = 0, absolute = false}` or `nil`;
+          setting a property to `nil` is equivalent to no override on that property
+        * `x`, `y` and `z` are in blocksize for position and scale (times ten), and in degrees for rotation
+        * `absolute`: If set to `false`, the override will be relative to the animated property:
+            * Transposition in the case of `position`;
+            * Composition in the case of `rotation`;
+            * Multiplication in the case of `scale`
+        * `interpolation_duration`: Time it takes (on the client, in seconds) to interpolate between the old & new value
+    * Not supported by older servers (< 5.6);
+      older clients (< 5.6) support only position & rotation
+      and will ignore interpolation & relativity entirely
```

Comment:
The ideas is that if modders use relative or interpolated values, they require their clients to be 5.7+. Other options include:

* Don't send update bone position commands to clients if they don't fully support them (probably the best option; what about interpolation though?)
* Add a new command
* Create a redundant bone position / bone override system to allow sending different values to older clients; this would be best for compat, but worst for the codebase and the effort required

Client visuals will not match what the server expects the client to see either way however, but not overriding the bones at all is probably more graceful than improperly overriding them.